internal/rpc/jsonrpc/methods.go

```diff
@@ -3317,6 +3318,42 @@ func (s *Server) sendToMultiSig(ctx context.Context, icmd interface{}) (interfac
 	return result, nil
 }
 
+// sendRawTransaction handles a sendrawtransaction RPC request by decoding hex
+// transaction and sending it to the network backend for propagation.
+func (s *Server) sendRawTransaction(ctx context.Context, icmd interface{}) (interface{}, error) {
+	cmd := icmd.(*dcrdtypes.SendRawTransactionCmd)
+	w, ok := s.walletLoader.LoadedWallet()
+	if !ok {
+		return nil, errUnloadedWallet
+	}
+
+	n, err := w.NetworkBackend()
+	if err != nil {
+		return nil, err
+	}
+
+	hexStr := cmd.HexTx
+	if len(hexStr)%2 != 0 {
+		hexStr = "0" + hexStr
+	}
+	serializedTx, err := hex.DecodeString(hexStr)
+	if err != nil {
+		return nil, rpcError(dcrjson.ErrRPCDecodeHexString, err)
+	}
+	msgtx := wire.NewMsgTx()
+	err = msgtx.Deserialize(bytes.NewReader(serializedTx))
```

Comment:
```
msgtx := wire.NewMsgTx()
err = msgtx.Deserialize(hex.NewDecoder(strings.NewReader(hexStr))
```

you can ignore and error for invalid hex strings (missing leading zeros), we don't produce them anywhere.